src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
@jglick I'm wondering if we actually need a new extension of `ProxyEnv` (let's call it `DefaultInvokerProxyEnv` because why not) that works the same as `ProxyEnv` except for returning `DefaultInvoker.INSTANCE` for `getInvoker()`, and then have, say, `CpsInternalFunction.invoke(...)` do `return super.invoke(new DefautInvokerProxyEnv(caller), ...)`? Or some such?